stock_picking_batch_extended/models/product_product.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2012-2014 Alexandre Fayolle, Camptocamp SA
+# Copyright 2023 FactorLibre - Boris Alias
```

Comment:
Review please the files that you haven't modify but added this.

Reply:
Fixed